ReactiveXComponent/Configuration/XCConfiguration.cs

```diff
@@ -0,0 +1,45 @@
+﻿
+namespace ReactiveXComponent.Configuration
+{
+    public class XCConfiguration
+    {
+        private readonly Parser.Parser _parser;
+        private readonly Tags _tags;
+
+        public XCConfiguration(Parser.Parser parser)
```

Comment:
Perhaps creating an interface IParser could be interesting
easier to test
To forget to write unit tests while coding implementations